.github/workflows/release.yaml

```diff
@@ -0,0 +1,152 @@
+              child.stderr.on("data", data => console.error(Buffer.from(data).toString()))
+              child.on("close", code => {
+                console.log(command, "exited with code", code)
+                if (code != 0) {
+                  core.setFailed(`Command ${command_str} exited with code ${code}`)
+                }
+              })
+            }
+
+            async function create_or_update_comment(identifier, comment_body) {
+              const comment_header = `<!-- ${identifier} -->\n`
+              const comment = comment_header + comment_body
+
+              const issue_number = context.issue.number
+              const owner = context.repo.owner
+              const repo = context.repo.repo
+
+              const existing_comments = (await github.rest.issues.listComments({
+                  issue_number, owner, repo
+              })).data
+
+              const comment_id = existing_comments.find(e => e.body?.startsWith(comment_header))?.id
+              const comment_obj = { owner, repo, body: comment }
+
+              if (comment_id) {
+                github.rest.issues.updateComment({ comment_id, ...comment_obj })
+              } else {
+                github.rest.issues.createComment({ issue_number, ...comment_obj })
+              }
+            }
+
+            async function create_github_release(crate_name, version) {
+              console.log("Creating a github release for", crate_name, version)
+
+              const { owner, repo } = context.repo
+              const tag_name = `${crate_name}-${version}`
+              const name = tag_name
+              const ref = `refs/tags/${tag_name}`
+              const sha = context.sha
+              const body = ""
+
+              await github.rest.git.createRef({ owner, repo, ref, sha })
+              await github.rest.repos.createRelease({ owner, repo, tag_name, name, body })
+            }
+
+            const is_dry_run = !is_merge_event
+            console.log("Dry run status is:", is_dry_run)
+
+            const crate_definitions = await globby("*/Cargo.toml")
+
+            const crates_to_publish = []
+            for (crate_definition of crate_definitions) {
+              console.log(`Processing ${crate_definition}`)
+
+              const data = toml.parse(fs.readFileSync(crate_definition, 'utf8'))
+              const name = data.package.name
+              const version_on_file = data.package.version
+              const version_on_crate_io = (await get_crate_details(name)).crate.max_version
+
+              if (version_on_file != version_on_crate_io) {
+                console.log("Crate", name, "need publishing")
+                crates_to_publish.push({ name, version_on_file, version_on_crate_io })
+
+                run_command_stream(`earthly -ci --org expressvpn --satellite wolfssl-rs --secret CARGO_REGISTRY_TOKEN +publish --PACKAGE=${name} --DRY_RUN=${is_dry_run}`)
```

Comment:
`-ci` should be `--ci`?

Reply:
Oh yes, thank you for catching that, will fix it 🙏